src/reporters/lang/en.js

```diff
@@ -132,6 +132,9 @@ const messages = {
   linkUnregistered: 'Unregistered $0.',
   linkUsing: 'Using linked module for $0.',
 
+  createInvalidBin: 'Invalid bin entry found in package $0.',
+  createMissingPackage: 'Package not found - this is probably an internal error, and should be reported on Github.',
```

Comment:
should this include a link to some specific issue tracker? "On GitHub" could mean anything.